merkle-tree-system/main.go

```diff
@@ -33,10 +34,7 @@ var config Configuration
 var tokenAuth *jwtauth.JWTAuth
 
 func init() {
-	tokenAuth = jwtauth.New("HS256", []byte("secret"), nil)
-
-	_, tokenString, _ := tokenAuth.Encode(jwt.MapClaims{"user_id": 123})
-	fmt.Printf("DEBUG: a sample jwt is %s\n\n", tokenString)
+	tokenAuth = jwtauth.New("HS256", []byte("secrettt"), nil)
```

Comment:
Do we still need this?

Reply:
JWT secret & algo are exported as env variables